tests/test_search_phrase_model.py

```diff
@@ -0,0 +1,16 @@
+# pylint: disable-all
+
+import torch
+
+from recommender.engine.rl_agent.models.history_model import HistoryModel
+
+
+def test_search_phrase_model_proper_shape():
+    SPE = 64  # service embedding dimension
+    X = 10  # length of a sequence
+    batch_size = 32
+
+    example_tensor = torch.rand(batch_size, X, SPE)
+    history_model = HistoryModel(SPE)
+
+    assert history_model(example_tensor).shape == torch.Size([batch_size, SPE])
```

Comment:
Looks like the copy of the `tests/test_history_model` file with changed function's name but without changing its body and imports.